ntttcp/ntttcp.Config.json

```diff
@@ -0,0 +1,61 @@
+{
+    "NtttcpDefault": {
+        "Iterations": 5,
+        "StartPort": 50000,
+        "tcp" : {
+            "BufferLen"     : [65536],
+            "Connections"   : [64],
+            "OutstandingIo" :[2, 16],
+            "Options"       : ""
+        },
+        "udp" : {
+            "BufferLen"     : [65536],
+            "Connections"   : [64],
+            "OutstandingIo" : null,
+            "Options"       : ""
+        },
+        "Warmup"   : 2,
+        "Cooldown" : 2,
+        "Time"     : 60

```

Comment:
Default runtime was 10. Please refer to the vars in the sampling config file / you can look back in the commit history. IIRC it was 1 iter and 10 sec as this was for sampling.